src/lib/atoms/KeyValue/KeyValue.tsx

```diff
@@ -1,18 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { FC, HTMLAttributes, ReactNode } from 'react';
 import classnames from 'classnames';
 
-// Helpers
+// Configs
+//@ts-ignore
 import { keyValueConfig } from 'configs';
 
 // Components
 import Icon from '../Icon';
 
 // Styles
-import './index.scss';
+import './KeyValue.scss';
+
+export interface IKeyValueProps extends HTMLAttributes<HTMLDivElement> {
+    /**
+     * Label for 'KeyValue'.
+     */
+    label: ReactNode;
+    /**
+     * Value for 'KeyValue'.
+     */
+    value: ReactNode;
+    /**
+     * External/Additional className that can be added to 'KeyValue' component.
+     */
+    className: string;
+    /**
+     * The way how the KeyValue should be displayed.
+     */
+    appearance: 'horizontal' | 'vertical';
+
+    icon: string;
+}
 
 // TODO need to refine where is using restProps
-function KeyValue({ label, value, className, icon, appearance, ...restProps }) {
+const KeyValue: FC<Partial<IKeyValueProps>> = ({
```

Comment:
Please remove Partial and in storybook KeyValue Docs you can see all types descriptions.
```suggestion
const KeyValue: FC<IKeyValueProps> = ({
```